src/pages/CardCollections/CardCollections.js

```diff
@@ -1,6 +1,223 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
+import { BsHeart, BsBookmark, BsChat } from 'react-icons/bs';
 
 export function CardCollections() {
-  return <div>CardCollections</div>;
+  const [photoCard, setPhotoCard] = useState([]);
```

Comment:
배열 형태임을 알 수 있게 복수형으로 네이밍하는 게 더 좋을 것 같습니다.
```suggestion
  const [photoCards, setPhotoCards] = useState([]);
```
혹은 뒤에 list를 붙여주시면 좋겠네요!
```suggestion
  const [photoCardList, setPhotoCardList] = useState([]);
```

Reply:
수정하였습니다.